Mathlib/Algebra/Associated.lean

```diff
@@ -530,6 +531,13 @@ theorem Associated.pow_pow [CommMonoid α] {a b : α} {n : ℕ} (h : a ~ᵤ b) :
   convert h.mul_mul ih <;> rw [pow_succ]
 #align associated.pow_pow Associated.pow_pow
 
+theorem Prod.associated_iff {M N : Type*} [Monoid M] [Monoid N] {x z : M × N} :
```

Comment:
You can use `~ᵤ` instead of `Associated`.

Reply:
I have made the modifications. Thank you!